dpnp/dpnp_iface_mathematical.py

```diff
@@ -126,7 +127,17 @@
 ]
 
 
-def abs(*args, **kwargs):
+def abs(
```

Comment:
I would propose to implement the same solution like numpy did:
```
abs = absolute
```
and to add a note to `dpnp.absolute` description that the `dpnp.abs` can be used as a shorthand for `dpnp.absolute`.